README.md

```diff
@@ -24,6 +24,7 @@ Somewhere in your code, you should setup the client:
 
 ;; Increment or derement a counter
 (statsd/increment "counter")
+(statsd/increment "counter" {:by 2.5 :sample-rate 3.3 :tags ["foo" "bar"]})
```

Comment:
I'd suggest removing `sample-rate` and `tags` and add a comment to describe what `by` is intended to do :)

Reply:
OK, changed it.